### DISCUSSION TITLE
v2.0.57 (Windows) - single-file.exe missing icon resource

### DISCUSSION THREAD

**Discussion by emp-00** [gildas-lormeau/single-file-cli#117] [General]
Not a huge deal, but a little bit disturbing - the latest release seems to be missing the usual icon. See screenshot (old vs new):
![image](https://github.com/user-attachments/assets/bfbc964c-3c90-4cbf-a736-072364abf12f)

In case this is "my fault", let me know. In "Properties" there's no option to add/change the icon?
![image](https://github.com/user-attachments/assets/43257fb3-137d-4317-89ea-bbe920fc8ae1)

**Comment by gildas-lormeau** [comment]
Thank you, I was not aware Deno removed the default icon. There is a new compile option in Deno to provide an `ico` file (on Windows), I'll try to use it in the next version.

**Reply by emp-00** [reply]
Appreciated! Thanks for the effort - it really looks more authentic if it has an icon, in my eyes - and it's much easier to spot single-file.exe in the the task manager. Looking forward to the next release. No hurries.

**Comment by gildas-lormeau** [comment]
The version `2.0.64` has an icon for Windows executable.
